.changes/next-release/bugfix-e737d87c-f84a-44f0-b403-5c692051e9d3.json

```
@@ -0,0 +1,4 @@
+{
+  "type" : "bugfix",
+  "description" : "AmazonQ client was replaced with netty client to add proxy server settings support"
```

users don't need to care that we're using netty
```suggestion
  "description" : "Respect IDE HTTP proxy server settings when using Amazon Q"
```

---

Done